### PR TITLE
updated synth-mapping.md

### PR DESCRIPTION
**Pull request by franticspider**
I've updated this file to match the new methodology for creating a synth mapping. I've also written a few modules for some of my own synths - plus updated/created modules for Streichfett.hs and SimpleSynth.hs - but I'm unable to check if these work!

### CODE REVIEW

**Comment by lennart** [comment]
yay @franticspider thx for your work, will need to review, I hope i can find time ;)

File: doc/midiable-functions.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+*Below is based on the tidal.lurk.org
```
Comment (lennart) [inline doc/midiable-functions.md]: did you mean http://tidalcycles.org ?

**Comment by franticspider** [comment]
The link in synth-mapping.md can be changed as you suggest.

the midiable-functions file is supposed to explain which of the 'core'
tidal functions can be used with midi. I'm not an experienced user of
tidal, but I've ralised we can't use functions like striate or smash for
example with midi because they are designed to work with samples. (I've
noticed that some tidal functions don't work with tidal-mid, so I was
making a list of what does and doesn't work.) To be honest, I'd forgotten
that I'd put that in my branch - you might not want it in master.

cheers,

Simon

**Comment by lennart** [comment]
@franticspider I try to find documentation for your jam jar drums, is this related [Jam Jar](https://vimeo.com/3745748) ? can you give us some description, so people know how to get or build one themselves?

I found the miniAtmegatron docs and will add just a link to the manual on which your mapping seems to be based, I'll add a separate pull request for the two of them

**Comment by lennart** [comment]
closing, see #29 & #27

**Comment by franticspider** [comment]
Hi Lennart,

No it's not that one! But it is a home-made drum machine that's based on
arduino and is held in a jam jar.... I guess it should be removed from the
master branch as it is an experimental synth.

Sorry for the confusion - I'm new to pull requests and I didn't really mean
you to pull the whole branch!

Simon (franticspider)

**Comment by lennart** [comment]
Ok, no worries!

Thanks for your effort!

You can simply create a lot of feature branch/pull requests with just the commits related to the change proposed in the description/title (and mostly keep things seperate that do not belong together), it helps to merge things since I cannot edit your branch directly ;)

**Pull request closed without merging**